internal/cf/binding.go

```diff
@@ -29,10 +44,19 @@ func (c *spaceClient) GetBinding(ctx context.Context, owner string) (*facade.Bin
 	if len(serviceBindings) == 0 {
 		return nil, nil
 	} else if len(serviceBindings) > 1 {
-		return nil, fmt.Errorf("found multiple service bindings with owner: %s", owner)
+		return nil, fmt.Errorf("found multiple service bindings with owner: %s", value)
```

Comment:
message "found multiple service bindings with owner" is only correct in case of empty bindingName